Tests/SentryTests/Integrations/Performance/IO/SentryFileIOTrackingIntegrationTests.swift

```diff
@@ -1,11 +1,14 @@
 import Foundation
 import Sentry
 import XCTest
-
+ 
+//This test is also executed under iOS-SwiftUITests
+//Thats why we need to keep it generic, without access
+//to any private part of the sdk
```

Comment:
`l`:
```suggestion
// This test is also executed under iOS-SwiftUITests, because
// GitHub Actions doesn't have simulators for iOS 11 and 10.
// That's why we need to keep it generic, without access
// to any private part of the SDK.
```